main.py

```diff
@@ -2,13 +2,18 @@
 from datetime import datetime
 
 
+with open("darabase.txt", "w") as f:    # just to create the file automatically 
+    f.write("2222:3333")
+
+
 def check_id_and_pin(user_id, user_pin):
-    if user_id == 2222 and user_pin == 3333:
-        return True
+    with open("database.txt", "r") as base:
```

Comment:
in the commits you can check how I did it before. 
But after I tried to think about the task a little bit more, and here is the result